landing-pages/site/static/.htaccess

```diff
@@ -1,9 +1,9 @@
 RedirectMatch Permanent ^/docs/(stable|1.10.10)/api(\.html)?$ "https://airflow.apache.org/docs/apache-airflow/$1/rest-api-ref"
 RedirectMatch Permanent ^/docs/(stable|1.10.10)/(cli|macros)(\.html)?$ "https://airflow.apache.org/docs/apache-airflow/$1/$2-ref"
 RedirectMatch Permanent ^/((_api|_images|_modules|_sources|_static|howto)/.*)$ "https://airflow.apache.org/docs/apache-airflow/stable/$1"
-RedirectMatch Permanent ^/((1.10.1|1.10.2|1.10.3|1.10.4|1.10.5|1.10.6|1.10.7|1.10.8|1.10.9|1.10.10)/.*)$ "https://airflow.apache.org/docs/apache-airflow/$1"
```

Comment:
So @ashb that > 1.10.10 was missing here was wrong, right?

Reply:
I think only versions 1.10.1 to 1.10.6.
https://github.com/apache/airflow-site/tree/f18ad1c764861e8b049391c7bf95f46a4adae916%5E

Reply:
I reverted this change as I only want to address the actual issue I found. I think this works as expected.

Only 1.10.14 and >= 2.0.0 was missing in the redirect from / to /docs/apache-airflow.

Reply:
WDYT? @mik-laj - pls re-review :)

Reply:
I don't think I need redirection for all files for 1.10.14 and 2.0.0 releases. We only need one redirection of one address for Airflow 2.0. See: 
https://github.com/apache/airflow-site/pull/362/files#r553838344

As for Airflow 1.10.14, we introduced a new link structure before this version was released.
![Screenshot 2021-01-08 at 10 40 20](https://user-images.githubusercontent.com/12058428/104000075-07a70100-519e-11eb-8a90-811e36eef2ee.png)

Reply:
Ahh. 1.10.14 is also affected by this problem, so we can use the following requests.
```
RedirectMatch Permanent ^/docs/(1.10.14|2\.[0-9.]+)/$ "https://airflow.apache.org/docs/apache-airflow/$1"
```